### PR TITLE
Simplify service_port_delta with environment variable

### PR DESCRIPTION
**Pull request by okurz**
Related progress issue: https://progress.opensuse.org/issues/153499

### CODE REVIEW

File: container/single-instance/Dockerfile [hunk 1]
```diff
@@ -17,11 +17,11 @@ LABEL org.opencontainers.image.created="%BUILDTIME%"
 # hadolint ignore=DL3037
 RUN zypper in -y openQA-single-instance openQA-bootstrap \
     qemu-arm qemu-ppc qemu-x86 qemu-tools sudo iputils os-autoinst-distri-opensuse-deps && \
-    zypper clean -a && \
-    sed -i -e 's/# service_port_delta =.*$/service_port_delta = 0/' /etc/openqa/openqa.ini
+    zypper clean -a
 EXPOSE 80 443 9526
 ENV skip_suse_specifics=1
 ENV skip_suse_tests=1
+ENV SERVICE_PORT_DELTA=0
```
Comment (kalikiana) [inline container/single-instance/Dockerfile:24]: Why is this not an OPENQA_ variable?

Comment (Martchus) [reply container/single-instance/Dockerfile:24]: Good question. I think it should be one.

Comment (okurz) [reply container/single-instance/Dockerfile]: Absolutely. Thank you! I only litterally replaced the variable name with an uppercase variant, haven't considered the prefix. Done

**Review by kalikiana — APPROVED** [review]

**Comment by okurz** [comment]
CI failure sounds related but I don't understand it:

```
[15:26:28] t/33-developer_mode.t .. 19/?     # Unexpected Javascript console errors, waiting for connection opened: [
    #   {
    #     level     => "SEVERE",
    #     message   => "http://localhost:9526/asset/3906633cf0/ws_console.js 8 WebSocket connection to 'ws://localhost:9528/liveviewhandler/tests/1/developer/ws-proxy' failed: Error during WebSocket handshake: Unexpected response code: 302",
    #     source    => "network",
    #     timestamp => 1707924445186,
    #   },
    # ]

    #   Failed test 'No unexpected js warnings'
    #   at /home/squamata/project/t/lib/OpenQA/Test/FullstackUtils.pm line 123.
    # Looks like you failed 1 test of 9.
[15:26:28] t/33-developer_mode.t .. 20/? 
#   Failed test 'resume test execution and 2nd tab'
#   at t/33-developer_mode.t line 340.
[15:26:28] t/33-developer_mode.t .. 21/? [15:26:28] t/33-developer_mode.t .. 22/? [15:26:28] t/33-developer_mode.t .. 23/? # Looks like you failed 1 test of 23.
                                          [15:26:28] t/33-developer_mode.t .. Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/23 subtests 
[15:28:29]
```

Could it be that I did not use the correct syntax in code?

**Comment by Martchus** [comment]
I also cross-checked the syntax and it really works (although brackets would probably not hurt for readability). The test also seems to generally work (also see log under artifacts) up to that point and `t/full-stack.t` is fully passing. The ports in the error message also look as they should and it also doesn't complain with "Connection refused". So this might even be an unrelated/sporadic issue.

**Comment by Martchus** [comment]
@Mergify-io rebase

**Pull request merged**